src/Carousel.js

```diff
@@ -0,0 +1,69 @@
+import React, {useState} from 'react';
+import styled from 'styled-components';
+import ButtonSlider from './ButtonSlider';
+
+const CarouselComponent = ({className, data}) => {
+    const [xPos, setXpos] = useState(0);
+
+    const moveCarousel = (direction) => {
+        if(direction === "prev" && xPos > -600){
+            setXpos(x => x - 200);
+        }
+     
+        if(direction === "next" && xPos < 600){
+            setXpos(x => x + 200);
+        }
+    };
+
+    return (
+    <div className={className}>
+        <div className="track" style={{transform: `translateX(${xPos}px)`, transitionDuration: "0.5s" }}>
+            {data.results.map(obj => {
```

Comment:
creating a component just for this can be useful in the future if you have more than one corousel